app/assets/javascripts/modules/external_users/claims/BlockHelpers.js

```diff
@@ -605,17 +605,18 @@ moj.Helpers.Blocks = {
       return $dom;
     };
 
-    this.setRadioState = function($dom, config) {
+    this.setRadioState = function ($dom, config) {
       // Car mileage visibility, radio checked & disabled values
       $dom.find('.fx-travel-mileage-car').css('display', config.car);
-      $dom.find('.fx-travel-mileage-car input').is(function() {
+      $dom.find('.fx-travel-mileage-car input').is(function () {
         $(this).prop('disabled', !config.carModel);
 
       });
 
       // Bike mileage visibility, radio checked & disabled values
       $dom.find('.fx-travel-mileage-bike').css('display', config.bike);
-      $dom.find('.fx-travel-mileage-bike input[type=radio]').is(function() {
+      console.log(config);
```

Comment:
Should we be outputting to console?

Reply:
Well spotted, we should never output to console.